### PR TITLE
convert egan data to numpy

### PR DESCRIPTION
**Pull request by etuleu**

### CODE REVIEW

File: worlds/pyegan/main.py [hunk 1]
```diff
@@ -1,11 +1,37 @@
...
+    env = EganEnv()
+
+    check_env(env)
+
+    model = PPO("MlpPolicy", env, verbose=1)
+    model.learn(total_timesteps=100)
+
+    log.info("training is done")
+
+    # obs = env.reset()
```
Comment (qemqemqem) [inline worlds/pyegan/main.py:25]: Do we need this commented out code still?

File: worlds/pyegan/EganEnv.py [hunk 2]
```diff
@@ -16,29 +17,30 @@ def __init__(self):
...
+            low=0, high=np.inf, shape=(HEIGHT, WIDTH), dtype=np.float
         )
 
         self.pyegan = PyEgan()
 
     def step(self, action):
         logger.info(f"step called action: %s", action)
         self.pyegan.wait_for_data(action)
-        return self.pyegan.obs
+        etensor = self.pyegan.obs["world"]
```
Comment (qemqemqem) [inline worlds/pyegan/EganEnv.py:33]: Unnecessary variable, only used once

File: worlds/pyegan/EganEnv.py [hunk 3]
```diff
@@ -16,29 +17,30 @@ def __init__(self):
...
+        return etensor_to_numpy(etensor), 1.0, False, {}
```
Comment (qemqemqem) [inline worlds/pyegan/EganEnv.py:34]: Add comments for these returns?

File: worlds/pyegan/EganEnv.py [hunk 4]
```diff
@@ -16,29 +17,30 @@ def __init__(self):
...
 
     def render(self, mode="human"):
         logger.info(f"render called mode: %s", mode)
 
     def reset(self):
         logger.info(f"reset called")
         self.pyegan.init()
-        return self.pyegan.obs
+        etensor = self.pyegan.obs["world"]
```
Comment (qemqemqem) [inline worlds/pyegan/EganEnv.py:42]: Same, variable not needed.

File: worlds/pyegan/PyEgan.py [hunk 5]
```diff
@@ -28,20 +31,22 @@ def step(self, observations, debug):
                 logger.info(f"step called obs: {k} size: {len(v.values)}")
 
         self.obs = observations
-        return self.action
+        res = {"VL": Action(discreteOption=self.action)}
```
Comment (qemqemqem) [inline worlds/pyegan/PyEgan.py:34]: Unnecessary variable

**Review by qemqemqem — APPROVED** [review]

File: worlds/pyegan/main.py (hunk 1 above)
Comment (etuleu) [reply worlds/pyegan/main.py:25]: deleted

File: worlds/pyegan/EganEnv.py (hunk 2 above)
Comment (etuleu) [reply worlds/pyegan/EganEnv.py:33]: fixed

File: worlds/pyegan/EganEnv.py (hunk 4 above)
Comment (etuleu) [reply worlds/pyegan/EganEnv.py:42]: fixed

File: worlds/pyegan/PyEgan.py (hunk 5 above)
Comment (etuleu) [reply worlds/pyegan/PyEgan.py:34]: fixed

**Pull request merged**